### PR TITLE
Rgb keyboard

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 1]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard toggle hotkey twice and note the result each time
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1311]:
```suggestion
1. Press the RGB keyboard toggle hotkey twice and note the result each time.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 2]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should disable the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1315]:
```suggestion
1. Pressing the button once should disable the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 3]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+1. Pressing the button again should re-enable the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1316]:
```suggestion
1. Pressing the button again should re-enable the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 4]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard toggle hotkey twice and note the result each time
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1339]:
```suggestion
1. Press the RGB keyboard toggle hotkey twice and note the result each time.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 5]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should disable the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1343]:
```suggestion
1. Pressing the button once should disable the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 6]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+1. Pressing the button again should re-enable the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1344]:
```suggestion
1. Pressing the button again should re-enable the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 7]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard color hotkey repeatedly until the keyboard cycles
+   through all color modes
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1368]:
```suggestion
   through all color modes.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 8]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should switch the keyboard color
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1372]:
```suggestion
1. Pressing the button once should switch the keyboard color.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 9]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+1. All color modes according to product documentation should be accessible
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1373]:
```suggestion
1. All color modes according to product documentation should be accessible.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 10]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard color hotkey repeatedly until the keyboard cycles
+   through all color modes
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1397]:
```suggestion
   through all color modes.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 11]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should switch the keyboard color
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1401]:
```suggestion
1. Pressing the button once should switch the keyboard color.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 12]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+1. All color modes according to product documentation should be accessible
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1402]:
```suggestion
1. All color modes according to product documentation should be accessible.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 13]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard brightness down hotkey and note the result
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1425]:
```suggestion
1. Press the RGB keyboard brightness down hotkey and note the result.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 14]
```diff
@@ -1286,3 +1286,225 @@ system.
...
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1452]:
```suggestion
1. Press the RGB keyboard brightness down hotkey and note the result.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 15]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should lower the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1456]:
```suggestion
1. Pressing the button once should lower the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 16]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard brightness up hotkey and note the result
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1479]:
```suggestion
1. Press the RGB keyboard brightness up hotkey and note the result.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 17]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should increase the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1483]:
```suggestion
1. Pressing the button once should increase the keyboard backlight.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 18]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Press the RGB keyboard brightness up hotkey and note the result
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1506]:
```suggestion
1. Press the RGB keyboard brightness up hotkey and note the result.
```

File: docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md [hunk 19]
```diff
@@ -1286,3 +1286,225 @@ system.
...
+
+**Expected result**
+
+1. Pressing the button once should increase the keyboard backlight
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md:1510]:
```suggestion
1. Pressing the button once should increase the keyboard backlight.
```

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

**Comment by mkopec** [comment]
@sulewskiprzemyslaw I fixed pre-commit's complaints and applied your suggested changes

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Comment by sulewskiprzemyslaw** [comment]
@macpijan @mkopec I have no further comments

**Pull request merged**